mt-kit/core/css/src/components/_dropdown.scss

```
@@ -54,10 +54,10 @@
     width: 0;
     top: 0;
     padding: 4.125rem var(--spacer-xx-small) var(--spacer-xx-small);
-    background-color: var(--dropdown-content-bg);
+    background-color: var(--color-mt-white);
     z-index: var(--z-index-feedback);
     right: 0;
-    border: $border-width-regular solid var(--color-mt-blue);
+    border-top: $border-width-regular solid var(--color-mt-beige-50);
```

according to figma i think this is should be
```suggestion
    border-top: $border-width-small solid var(--color-mt-mellombeige);
```